package/libyui.changes

```diff
@@ -8,7 +14,6 @@ Wed Oct 13 09:39:18 UTC 2021 - Stefan Hundhammer <shundhammer@suse.com>
 
 -------------------------------------------------------------------
 Tue Aug 24 15:30:52 UTC 2021 - Ladislav Slezák <lslezak@suse.cz>
-
```

Comment:
This change is OK, but I'm not sure if some OBS check will not complain about "not incremental" changes.... :thinking: 
I'd rather revert it.